src/main/java/org/mule/extension/sftp/internal/command/SftpCommand.java

```diff
@@ -82,13 +82,19 @@ public SftpFileAttributes getFile(String filePath) {
 
   protected SftpFileAttributes getFile(String filePath, boolean requireExistence) {
     URI uri = resolvePath(normalizePath(filePath));
+    if (LOGGER.isTraceEnabled()) {
```

Comment:
Ditto for isTraceEnabled

Reply:
I think this isn't a significant overhead here. And it's generally used in this way.